common/model-views.cpp

```diff
@@ -2014,24 +2014,27 @@ namespace rs2
     {
         // checking format
         bool is_cal_format = false;
-        for (auto it = stream_enabled.begin(); it != stream_enabled.end(); ++it)
+        if (dev.supports(RS2_CAMERA_INFO_PRODUCT_ID) && !strcmp(dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "0B5B"))
```

Comment:
1. Please add a comment about what `0B5B` means, magic numbers should not be used without explaining.
I would maybe add a `static const string D405_PID = "0B5B"` for all D405 usage in this file

2. Please explain (add a comment) why D405 should always return false here

Reply:
Done